docker/Dockerfile-ee

```diff
@@ -0,0 +1,82 @@
+#
+# Run bash in this container from main project folder and export package versions to Docker/requirements.txt:
+# >>> docker run --rm -it --shm-size=2g --net=host -v "$PWD":/home/ghsci/work globalhealthyliveablecities/global-indicators /bin/bash
+# >>> pip list --format=freeze > ./Docker/requirements.txt
+#
+# Push to docker hub
+# docker login
+# docker tag globalhealthyliveablecities/global-indicators globalhealthyliveablecities/global-indicators:latest
+# docker tag globalhealthyliveablecities/global-indicators globalhealthyliveablecities/global-indicators:yyyy-mm-dd
+# docker push --all-tags globalhealthyliveablecities/global-indicators
+#
+# Stop/delete all local docker containers/images:
+# >>> docker stop $(docker ps -aq)
+# >>> docker rm $(docker ps -aq)
+# >>> docker rmi $(docker images -q) --force
+##############################################################################
+
+FROM continuumio/miniconda3:latest AS build
+LABEL maintainer="Global Healthy Liveable City Indicator Study Collaboration Group"
+LABEL url="https://github.com/global-healthy-liveable-cities/global-indicators"
+
+COPY environment-ee.yml .
+
+# add root/bin to path so that tex commands can be run from container
+ENV PATH=$PATH:/root/bin
+
+# Install conda pack
+RUN conda config --set show_channel_urls true && \
+    conda config --set channel_priority strict && \
+    conda config --prepend channels conda-forge && \
+    conda update --yes -n base conda && \
+    conda install -c conda-forge conda-pack && \
+    conda env create -n env -f environment.yml && \
+    conda pack -n env -o env.tar.gz && \
+    conda list -n env --explicit > spec-file.txt
+
+
+# =============================================================================
+# Runtime environment
+FROM debian:stable-slim AS runtime
+
+# Install the environment pack
+COPY --from=build /env.tar.gz .
+RUN mkdir /env && \
+    tar xvzf /env.tar.gz -C /env && \
+    rm -rv /env.tar.gz && \
+    /env/bin/python /env/bin/conda-unpack && \
+    useradd -u 8877 ghsci && \
+    # Install tools for using OpenStreetMap and constructing uharfbuzz for text shaping in pdfs
+    apt-get update && apt-get install -y gcc g++ git osm2pgsql osmctools && \
+    # Install the Google Cloud SDK
+    apt-get update && apt-get install -y apt-transport-https ca-certificates gnupg curl && \
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | gpg --dearmor -o /usr/share/keyrings/cloud.google.gpg && \
+    echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
+    apt-get update && apt-get install -y google-cloud-cli
+
+ENV PATH="/env/bin/:$PATH"  
+ENV PROJ_LIB="/env/share/proj"
+
+# Install additional Python packages including earthengine-api and geemap
+RUN pip install uharfbuzz && \
+    pip install -e git+https://github.com/wklumpen/gtfs-lite.git#egg=gtfs-lite && \
```

Comment:
This still looks to be based on a not-current version of the Dockerfile; I recommend we base off the latest which simplfiies these things (these  pip installed packages are either not necessary and dropped, or incorporated in environment-ee.yml).   Can earthengine-api and geemap be installed using pip within environment-ee.yml in a similar way?  If so, we should do that to keep the environment specifics in the environment file.